Solutions/Cybersixgill-Actionable-Alerts/Playbooks/DeleteCybersixgillAlert/azuredeploy.json

```diff
@@ -0,0 +1,437 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Delete-Cybersixgill-Alert",
+        "description": "This playbook will delete Alert on Cybersixgill portal when resective Incident is deleted in Microsoft Sentinel",
+        "prerequisites": "",
+        "prerequisitesDeployTemplateFile": "",
+        "lastUpdateTime": "2022-11-29T18:18:05Z",
+        "entities": [],
+        "tags": ["Incident", "Sync"],
+        "support": {
+            "tier": "community",
+            "armtemplate": "Generated from https://github.com/Azure/Azure-Sentinel/tree/master/Tools/Playbook-ARM-Template-Generator"
+        },
+        "author": {
+            "name": "Loginsoft"
+        }
+    },
+    "parameters": {
+        "PlaybookName": {
+            "defaultValue": "DeleteCybersixgillAlert",
+            "type": "string"
+        },
+        "Keyvault Name": {
+            "type": "string",
+            "defaultValue":"",
+            "metadata": {
+                "description": "Enter value for Keyvault Name"
+            }
+        },
+        "Cybersixgill Client ID": {
+            "type": "string",
+            "metadata": {
+                "description": "Enter value for Cybersixgill Client ID"
+            }
+        },
+        "Cybersixgill Client Secret": {

```

Comment:
Cybersixgill Client Secret , this will confuse the customer , by looking at name customer will save actual secret key here , kindly mention "Cybersixgill Client Secret Key name".

do the same with "Cybersixgill Client ID" should be replace with "Cybersixgill Client ID key name" 

Then update the playbook parameters in workflow accordingly and readme file as well